compiler/src/iree/compiler/Codegen/Common/GPU/WorkgroupReordering.cpp

```diff
@@ -68,6 +68,95 @@ makeSwizzledIds(Location loc, OpBuilder b, Value workgroupIdX,
   return {swizzledIdX, swizzledIdY};
 }
 
+// reoredering to make workgroup ids move slowly between chiplet groups
```

Comment:
```suggestion
// Reordering to make workgroup ids move slowly between chiplet groups.
```

Could you also give an example? IE pick some topology and show how the math works out.

Also say what the return value is.